### PR TITLE
Allow adding custom HTTP headers to bosh communications

### PR DESCRIPTION
**Pull request by mhariri**

### CODE REVIEW

**Comment by mhariri** [comment]
The build should succeed after https://github.com/igniterealtime/jbosh/pull/13 is merged.

File: smack-bosh/build.gradle [hunk 1]
```diff
@@ -4,5 +4,5 @@ This API is considered beta quality."""
 
 dependencies {
 	compile project(':smack-core')
-	compile 'org.igniterealtime.jbosh:jbosh:[0.9,0.10)'
+	compile 'org.igniterealtime.jbosh:jbosh:[0.10,0.11)'
```
Comment (Flowdalic) [inline smack-bosh/build.gradle]: I was actually planning to add support for HTTP headers to jbosh 0.9.1, so the version range would become `[0.9.1,0.10)`. (And that are the things I wanted to discuss with you in IRC ;))

**Comment by mhariri** [comment]
Changed it to 0.9.1

**Comment by Flowdalic** [comment]
https://github.com/igniterealtime/jbosh/pull/13 merged and jbosh 0.9.1 released. Waiting for Maven Central mirrors to sync.

**Comment by Flowdalic** [comment]
Applied as https://github.com/igniterealtime/Smack/commit/c83d717a26544eb9c355a0ca932a3cca2867c88f

**Pull request closed without merging**